src/components/Footer/Footer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Navigation from '../Navigation/Navigation'
+
+export default function Footer() {
+  return (
+    <footer role="contentinfo">
```

Comment:
I don't understand this role? If this is just there for selection during testing, maybe go with a data-js attribute, or simply the select the footer as a type?

Reply:
You are right. For access. the footer tag is sufficient. Will remove it.